boards/xmc-2go/include/periph_conf.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2015 Sebastian Sontberg <sebastian@sontberg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_xmc2go
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the xmc-2go board
+ *
+ * @author      Sebastian Sontberg <sebastian@sontberg.de>
+ */
+
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
+
+#include "cpu.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name RTC configuration
+ * @{
+ */
+#define RTC_NUMOF          (1U)
+#define RTC_IRQ_PRIO       CPU_DEFAULT_IRQ_PRIO
+#define RTC_DIVIDER        (0x7fff)
+/** @} */
+
+/**
+ * @name PRNG configuration
+ * @{
+ */
+#define RANDOM_NUMOF       (1U)
+/** @} */
+
+/**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF          (2U)
+#define ADC_0_EN           1
+#define ADC_1_EN           0
+#define ADC_MAX_CHANNELS   8
+/** @} */
+
+/**
+ * @name Timer configuration
+ * @{
+ */
+#define TIMER_NUMOF        (1)
+#define TIMER_0_EN         (1)
+#define TIMER_0_CHANNELS   (2)
+#define TIMER_0_MAX_VALUE  (0xffffffff)
+#define TIMER_IRQ_PRIO     CPU_DEFAULT_IRQ_PRIO
+
+#define XTIMER_BACKOFF     (40)
+#define XTIMER_ISR_BACKOFF (30)
+/** @} */
+
+/**
+ * @name UART configuration
+ * @{
+ */
+#define UART_NUMOF         (1U)
+#define UART_0_EN          (1)
+#define UART_IRQ_PRIO      CPU_DEFAULT_IRQ_PRIO
+
+static const uart_conf_t uart_dev[] = {
+    {
+        .usic = USIC0_CH0,     // USIC
+        .tx = GPIO(P2, 1),     // TX pin
+        .rx = GPIO(P2, 2),     // RX pin
+        .dsel = 6,             // configure input control register 0
+        .asel = GPIO_ALT_OUT_6 // to use data input DXnG (see page 451)
```

Comment:
This is Arduino style labeling. The idea is: TX is the sending part of the connected shield or USB Bridge and requires a connection to the input of the microcontroller (XMC), hence TX > P1.3. Anyway, it can be misunderstood easily.
The TX/RX mismatch is a common trouble. With the XMC1000 microcontrollers, the TX and RX function can be swapped by software configuration.